### PR TITLE
added note about laziness of operations to statistical functions

### PR DESCRIPTION
**Pull request by corinnebosley**
Only two functions to clarify here.  This is a small one to start us off.

### CODE REVIEW

**Comment by corinnebosley** [comment]
@valeriupredoi @ledm How's this format for you?

**Comment by ledm** [comment]
Good one, thanks!

**Comment by corinnebosley** [comment]
@pp-mo @bjlittle @lbdreyer I've got a whole load of these PRs coming up, if somebody could spare the time to check them over as they come up I'd be really grateful.

**Comment by valeriupredoi** [comment]
music to my ears :grin: cheers @corinnebosley :beer:

File: lib/iris/analysis/stats.py [hunk 1]
```diff
@@ -31,6 +31,8 @@
 def _ones_like(cube):
     """
     Return a copy of cube with the same mask, but all data values set to 1.
+
+    The operation is non-lazy.
```
[the remark this answers is not in the document]
Comment (rcomer) [reply lib/iris/analysis/stats.py]: I guess this note would be useful for the next developer who looks at the code, wanting to make the public function lazy?

**Comment by bjlittle** [comment]
@corinnebosley Thanks.

**Pull request merged**